index.html

```diff
@@ -31,13 +32,213 @@ <h2 id="job-title">I am a Software Developer</h2>
+                    <li><a href="#"><img class="icon-interactions" src="images/linkedin.png" alt="union-icon"></li></a>
+                    <li><a href="#"><img class="icon-interactions" src="images/hand.png" alt="hand"></li></a>
+                    <li><a href="#"><img class="icon-interactions" src="images/twitter.png" alt="twitter bird"></li></a>
+                    <li><a href="#"><img class="icon-interactions" src="images/Vector.png" alt="M-sign"></li></a>
                 </ul> 
             </section>
+            <section class="work">
+                <div>
+                    <div class="recent-work">
+                        <h2 id="work-intro">My Recent Work</h2>
+                        <a id="yoga-image" href=""><img src="images/Img Placeholder-1.png" alt="yoga girl"></a>
+                    </div>
+                    <div class="multi-post-div">
+                        <h2 id="multi-post-title" >Multi-Post Stories</h2>
+                        <div id="work-description">
+                            <p id="work-font">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        </div>
+                        <div class="button-div">
+                            <ul class="button-list">
+                                <li><a href="#"><button class="multi-post-button">css</button></a></li>
+                                <li><a href="#"><button class="multi-post-button">html</button></a></li>
+                                <li><a href="#"><button class="multi-post-button">bootstrap</button></a></li>
+                                <li><a href="#"><button class="multi-post-button">Ruby</button></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <a href="#"><button class="first-see-project-button">See Project</button></a>
+                </div>
+                <div class="project-1">
+                    <img src="images/Img Placeholder-2.png" alt="printed-magazine">
+                    <div class="floating">
+                        <div>
+                            <h3 class="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <ul class="project-1-buttons-ul">
+                            <a href="#"><button class="project-1-buttons">html</button></a>
+                            <a href="#"><button class="project-1-buttons" >bootstrap</button></a>
+                            <a href="#"><button class="project-1-buttons">Ruby</button></a>
+                        </ul>
```

Comment:
- [ ] Kindly use the proper unorder list syntax, `ul and li` go the same as a parent and child structure like the example below :+1: .

Example
```
  <ul>
      <li><a href="#">html</a></li>
      <li><a href="#">css</a></li>
      <li><a href="#">bootstrap</a></li>
  </ul>
```